DependencyInjection/Configuration.php

```diff
@@ -50,6 +50,12 @@ public function getConfigTreeBuilder()
                         ->scalarNode('password')->end()
                     ->end()
                 ->end()
+                ->arrayNode('html5validation')
+                    ->children()
+                        ->scalarNode('url')->end()
+                        ->arrayNode('ignores_extract')->end()
```

Comment:
It was needed because of this line: https://github.com/liip/LiipFunctionalTestBundle/pull/211/files#diff-da25f5e5e9255b14c3d3d96edd20bb24R154

It only define nodes already used in the tests: https://github.com/alexislefebvre/LiipFunctionalTestBundle/blob/html5-validator/Test/Html5WebTestCase.php#L69